### PR TITLE
Rename cpl to cpflow

### PR DESCRIPTION
**Pull request by Seifeldin7**
### Summary
- Renamed instances of `cpl` gem to `cpflow` as `cpl` gem is deprecated and updated some commands.

- - -
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react-webpack-rails-tutorial/595)

## Summary by CodeRabbit

- **Documentation**
	- Updated all references from `cpl` to `cpflow` in configuration files, documentation, and setup instructions.
	- Enhanced deployment commands to reflect the new `cpflow` tool.
	- Adjusted installation instructions to use `cpflow` gem version `4.0.0`.
	- Migrated example commands and environment variable references to align with `cpflow`.

### CODE REVIEW

File: .github/actions/deploy-to-control-plane/action.yml [hunk 1]
```diff
@@ -62,9 +62,9 @@ runs:
         # Run database migrations (or other release tasks) with the latest image,
         # while the app is still running on the previous image.
         # This is analogous to the release phase.
-        cpl run:detached './.controlplane/release_script.sh' -a ${{ inputs.app_name }}  --image latest
```
Comment (Seifeldin7) [inline .github/actions/deploy-to-control-plane/action.yml:65]: run:detached has been deprecated

File: .controlplane/readme.md [hunk 2]
```diff
@@ -100,22 +100,22 @@ After committing code, you will update your deployment of `react-webpack-rails-t
...
 # Assuming you have already set APP_NAME env variable to react-webpack-rails-tutorial
 # Build and push new image with sequential image tagging, e.g. 'react-webpack-rails-tutorial:1', then 'react-webpack-rails-tutorial:2', etc.
-cpl build-image -a $APP_NAME
+cpflow build-image -a $APP_NAME
 
 # Run database migrations (or other release tasks) with latest image,
 # while app is still running on previous image.
 # This is analogous to the release phase.
-cpl runner rails db:migrate -a $APP_NAME --image latest
+cpflow run -a $APP_NAME --image latest -- rails db:migrate
```
Comment (Seifeldin7) [inline .controlplane/readme.md:108]: Updated according to the gem documentation.

**Comment by rameziophobia** [comment]
@justin808 @Seifeldin7 I upgraded cpflow to v4 and it works as expected. I tested the deployment [here](https://rails-3k23yt62491bp.cpln.app/simple)

I will squash and merge this to master and then update #597 to use the new base

**Review by justin808 — COMMENTED** [review]
Looks good

**Pull request merged**